Add speedometer

Valkama1 opened the pull request:
This pull request closes #44 

A speedometer has been added to the top of the camera feed according to the issue specifications. 
 - Code in sketch for publishing odometer reading to odometer mqtt topic
 - Added a text view to the top of the camera view
 - Changed to a relative layout to allow overlapping of elements
 - Added logic for converting and displaying the odometer speed in kmh in the text view

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change

Commit:
Add speedometer

Lindet94 committed:
#49 Add crash button for demo

Lindet94 committed:
Add Felix email as receiver

Lindet94 commented on arduino/smartcar/vehicle_config.json:
```diff
@@ -1,83 +1,83 @@
-   }
-}
\ No newline at end of file
+// {
```
Some commented out code here. Do we keep it?

Valkama1 replied:
Oh I forgot about this. I think we can remove the file since it was only used to make the emulator car into a tank. I'll remove it

Valkama1 replied:
Deleted.

Lindet94 commented on android/app/src/main/java/safetyfirst/androidapp/safetyfirstcontroller/MainActivity.java:
```diff
@@ -235,6 +240,14 @@ public void messageArrived(String topic, MqttMessage message) throws Exception {
+                        TextView speedometer = (TextView)findViewById(R.id.speedometer);
+
+                        double speedMS = Double.parseDouble(message.toString());
+                        double speedKMH = Math.round((speedMS * 10)*10.0)/10.0;
```
Correct calculation here?

Valkama1 replied:
It wasn't, must have been pretty stressed when I did that.🤣 
Fixed now

Lindet94 reviewed (changes requested):
Looks good! Just a couple of small things to look over before merging. Check comments :)

Valkama1 committed:
Updated kmh calculation

Updated incorrect kmh calculation for the speedometer

Valkama1 committed:
Delete unused file

This file was used to make the car in the emulator look like a tank. This i irrelevant for our project so I removed it.
